src/Microsoft.Extensions.Configuration.Abstractions/IConfiguration.cs

```diff
@@ -36,5 +36,6 @@ public interface IConfiguration
         IEnumerable<IConfigurationSection> GetChildren();
 
         IChangeToken GetReloadToken();
+        IConfigurationRoot Root { get; }
```

Comment:
I feel like `GetReloadToken()` as a method is better because you'll always been accessing the token in a 'loop'. You never really want to cache the result.